jest.config.js

```diff
@@ -11,6 +11,24 @@ module.exports = {
     '@stencil/core/testing': '<rootDir>/testing/index.js',
     '@utils': '<rootDir>/src/utils',
   },
+  coverageDirectory: './test-results/',
```

Comment:
One non-blocking suggestion - I think `./coverage/` may be more apt here. We have a lot of `test*` directories today, naming this to something that helps differentiate it may make the top level directory of the repo a little more grok-able.

Reply:
Done!